socorro/external/es/super_search_fields.py

```diff
@@ -14,6 +14,19 @@
 from socorrolib.lib import datetimeutil, external_common
 
 
+ES_CUSTOM_ANALYZERS = {
+    "analyzer": {
+        "semicolon_keywords": {
+            "type": "pattern",
+            "pattern": ";"
+        }
```

Comment:
actually. I don't care. but I noticed sometimes you tidy these things up so perhaps you would appreciate the finding.